internal/handlers/telegram/documentHandler.go

```diff
@@ -0,0 +1,39 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+)
+
+
+type DocumentObject struct {
+	EnableLink bool
```

Comment:
WIth the v2.0 iteration of TeleIRC, I would like to move away from enabling any sort of boolean for sending documents.